cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/index.tsx

```diff
@@ -65,6 +65,12 @@ const styles = (theme) => {
       paddingLeft: '10px',
       fontWeight: 'bold',
       fontSize: '1rem',
+      overflow: 'hidden',
```

Comment:
is this still required?

Reply:
The ellipses aren't there unless I have overflow: 'hidden" - are you seeing something different?